index.html

```diff
@@ -5132,6 +5132,9 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MAY be used with <dfn>external proof</dfn>.
```

Comment:
The improved suggestion from @msporny in the other PR:

```suggestion
    This media type can be used with credentials secured using an <dfn>external proof</dfn>.
```